_scripts/evaluate_doc_retrieval__model_llm__2024-03-15T17:58:10.862451.csv

```diff
@@ -0,0 +1,6 @@
+test_id,query,state_filter_expected,county_filter_expected,commodity_filter_expected,doc_category_filter_expected,retrieved_doc1_expected,retrieved_doc2_expected,retrieved_doc3_expected,state_filter_actual,county_filter_actual,commodity_filter_actual,doc_category_filter_actual,retrieved_doc1_actual,retrieved_doc2_actual,retrieved_doc3_actual,nb_of_FindDocs_nodes_actual,state_filter_match,county_filter_match,commodity_filter_match,doc_category_filter_match,retrieved_doc1_match,retrieved_doc2_match,retrieved_doc3_match,eval_score
```

Comment:
On devrait gitignore ces fichiers non?

Reply:
I wanted to keep under git the current results, as a reference to what is expected... so I would keep them there if you don't mind.